integration_tests/commands/setup.go

```diff
@@ -67,7 +69,12 @@ func FireCommand(conn net.Conn, cmd string) interface{} {
 	args := testutils.ParseCommand(cmd)
 	_, err = conn.Write(clientio.Encode(args, false))
 	if err != nil {
-		log.Fatalf("error %s while firing command: %s", err, cmd)
+		slog.Error(
```

Comment:
similar issue as https://github.com/DiceDB/dice/pull/584#discussion_r1759977990, have used a global logger for now